pkg/datapath/iptables/iptables.go

```diff
@@ -1835,6 +1891,37 @@ func (m *Manager) addCiliumNoTrackEncryptionRules() (err error) {
 	return nil
 }
 
+func (m *Manager) addCiliumAcceptTunnelRules() (err error) {
+	port := m.sharedCfg.TunnelPort
+
+	if !m.sharedCfg.TunnelingEnabled || port == 0 {
+		return nil
+	}
+
+	cmd := []string{
+		"-t", "filter",
+		"-A", ciliumOutputChain,
+		"-p", "udp",
+		"--dport", strconv.Itoa(int(port)),
+		"-m", "comment", "--comment", "cilium: ACCEPT for tunnel traffic",
+		"-j", "ACCEPT",
+	}
+
+	if m.sharedCfg.EnableIPv4 {
```

Comment:
```suggestion
	if m.sharedCfg.EnableIPv4 && ip4tables != nil {
```

(Purely from what we do in the NOTRACK path)

Reply:
ip4tables global doesn't exist anymore. maybe i forgot to adjust it

Reply:
i dont know how the compiler isnt complaining, but ip4tables isnt set anywhere

Reply:
oh right, those 2 are passed by the caller (https://github.com/cilium/cilium/pull/42356/files#diff-465a6b3f4ab9e1e94fe39e5aefe81c41eaa1dc99c66500161072e564029c2f51R641)